src/modules/layout/containers/PermissionSnackbarContainer.js

```diff
@@ -98,18 +100,47 @@ class PermissionSnackbarContainer extends React.Component<PropsType, StateType>
     return currentRoute.key === 'Landing'
   }
 
+  cityRoute = (): boolean => {
+    const { navigation } = this.props
+    const { index, routes } = navigation.state
+    const currentRoute = routes[index]
+
+    return currentRoute.key === 'CityContent'
+  }
+
   dashboardRoute = (): boolean => {
     const { navigation } = this.props
     const { index, routes } = navigation.state
     const currentRoute = routes[index]
 
-    return currentRoute.key === 'CityContent' && currentRoute.routes[currentRoute.index]?.routeName === 'Dashboard'
+    return this.cityRoute() && currentRoute.routes[currentRoute.index]?.routeName === 'Dashboard'
+  }
+
+  shouldShowCoronaSnackbar = () => {
+    return this.state.showCoronaSnackbar && (this.landingRoute() || this.cityRoute())
```

Comment:
I thought we only want to show it in the cityRoute?

Reply:
Not that I know of. At least not if it is required by google. Because if it is required by google, it should be shown as soon as possible. Or did I miss something here?